### ISSUE TITLE
Reviewed papers(tmp)

### ISSUE THREAD

**Issue by doublejy715** [doublejy715/Paper_review#34]
No | Title | Conference | Year | Link | Code | Skimming | Summary |  
------------ | ------------- |----------|----------|----------|----------|----------|----------|  
1 | Analyzing and Improving the Image Quality of StyleGAN(StyleGAN2) |CVPR |2019 |[arXiv](https://arxiv.org/abs/1912.04958)|[Code](https://github.com/NVlabs/stylegan2)||[Summary](https://github.com/doublejy715/Paper_review/issues/3)|  
2 | Training Generative Adversarial Networks with Limited Data(StyleGAN2-ada) | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2006.06676v1)|[Code](https://github.com/NVlabs/stylegan2-ada-pytorch)||[Summary]() |  
3 | A Style-Based Generator Architecture for Generative Adversarial Networks(StyleGAN) | CVPR | 2019 | [arXiv](https://arxiv.org/abs/1812.04948) | [Code](https://github.com/NVlabs/stylegan) || [Summary](https://github.com/doublejy715/Paper_review/issues/2)|  
4 | Image2StyleGAN: How to Embed Images Into the StyleGAN Latent Space? | CVPR | 2019 | [arXiv](https://arxiv.org/abs/1904.03189) | None || [Summary](https://github.com/doublejy715/Paper_review/issues/4) |
5 | Arbitrary Style Transfer in Real-time with Adaptive Instance Normalization(AdaIN) | CVPR | 2017 | [arXiv](https://arxiv.org/abs/1703.06868) | [Code](https://github.com/xunhuang1995/AdaIN-style) || [Summary](https://github.com/doublejy715/Paper_review/issues/5) |
7 | InterFaceGAN: Interpreting the Disentangled Face Representation Learned by GANs | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2005.09635) | [Code](https://github.com/genforce/interfacegan) | |[Summary](https://github.com/doublejy715/Paper_review/issues/7)|
8 | Fine-Tuning StyleGAN2 For Cartoon Face Generation | CVPR | 2021 | [arXiv](https://arxiv.org/abs/2106.12445) | [Code](https://github.com/happy-jihye/Cartoon-StyleGan2) ||[Summary](https://github.com/doublejy715/Paper_review/issues/8) |
9 | Encoding in Style: a StyleGAN Encoder for Image-to-Image Translation(pSp) | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2008.00951v1) | [Code](https://github.com/eladrich/pixel2style2pixel) | |[Summary](https://github.com/doublejy715/Paper_review/issues/10) |
10 | Animating Arbitrary Objects via Deep Motion Transfer | CVPR | 2019 | [arXiv](https://arxiv.org/abs/1812.08861) | [Code](https://github.com/AliaksandrSiarohin/monkey-net) || [Summary](https://github.com/doublejy715/Paper_review/issues/11) |
11 | First Order Motion Model for Image Animation | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2003.00196)| [Code](https://github.com/AliaksandrSiarohin/first-order-model) | |[Summary](https://github.com/doublejy715/Paper_review/issues/12) |
12 | Controllable Person Image Synthesis with Attribute-Decomposed GAN(ADGAN) | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2003.12267)| [Code](https://github.com/menyifang/ADGAN) | |[Summary](https://github.com/doublejy715/Paper_review/issues/13) |
13 | StyleFlow: Attribute-conditioned Exploration of StyleGAN-Generated Images using Conditional Continuous Normalizing Flows | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2008.02401) | [Code](https://github.com/RameenAbdal/StyleFlow) || [Summary](https://github.com/doublejy715/Paper_review/issues/14) |
14 | Closed-Form Factorization of Latent Semantics in GANs | CVPR | 2019 | [arXiv](https://arxiv.org/abs/2007.06600v1) | [Code](https://paperswithcode.com/paper/closed-form-factorization-of-latent-semantics) | |[Summary](https://github.com/doublejy715/Paper_review/issues/16) |
15 | StyleSpace Analysis : Disentangled Controls for StyleGAN Image Generation | CVPR | 2020 |[arXiv](https://arxiv.org/abs/2011.12799v1)| [Code](https://github.com/betterze/StyleSpace) | |[Summary](https://github.com/doublejy715/Paper_review/issues/17) |
16 | StyleMapGAN: Exploiting Spatial Dimensions of Latent in GAN for Real-time Image Editing | CVPR | 2021 | [arXiv](https://arxiv.org/abs/2104.14754) | [Code](https://github.com/naver-ai/StyleMapGAN) || [Summary](https://github.com/doublejy715/Paper_review/issues/18) |
17 | Freeze the Discriminator: a Simple Baseline for Fine-Tuning GANs V1 | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2002.10964v1)| [Code](https://github.com/sangwoomo/freezeD)||[Summary](https://github.com/doublejy715/Paper_review/issues/19)|
18 | pix2pix : Image-to-Image Translation with Conditional Adversarial Networks | CVPR | 2016 | [arXiv](https://arxiv.org/abs/1611.07004v1) | [Code](https://github.com/phillipi/pix2pix) | |[Summary](https://github.com/doublejy715/Paper_review/issues/22) |
19 | e4e : Designing an Encoder for StyleGAN Image Manipulation | CVPR | 2021 | [arXiv](https://arxiv.org/abs/2102.02766) | [Code](https://github.com/omertov/encoder4editing) | [Skimming]() | [Summary]() |
20 | In-domain gan inversion for real image editing | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2004.00049) | [Code](https://github.com/genforce/idinvert)| [Skimming](https://github.com/doublejy715/Paper_review/issues/24)| [Summary]() |
21 | StyleClip : Text-Driven Manipulation of StyleGAN Imagery | CVPR | 2021 | [arXiv](https://arxiv.org/abs/2103.17249) | [Code](https://github.com/orpatashnik/StyleCLIP)| [Skimming](https://github.com/doublejy715/Paper_review/issues/25)| [Summary](https://github.com/doublejy715/Paper_review/issues/26) |
22 | DeepFaceEditing : Deep Face Generation and Editing with Disentangled Geometry and Appearance Control | CVPR | 2021 | [arXiv](https://arxiv.org/abs/2105.08935) | [Code](http://geometrylearning.com/DeepFaceEditing/)| [Skimming](https://github.com/doublejy715/Paper_review/issues/27)| [Summary](https://github.com/doublejy715/Paper_review/issues/28) |
23 | StyleMapGAN: Exploiting Spatial Dimensions of Latent in GAN for Real-time Image Editing | CVPR | 2021 | [arXiv](https://arxiv.org/abs/2104.14754v1) | [Code](https://github.com/naver-ai/StyleMapGAN)| [Skimming](https://github.com/doublejy715/Paper_review/issues/29)| [Summary]() |
24 | DCGAN:Unsupervised Representation Learning with Deep Convolutional Generative Adversarial Networks | CVPR | 2015 | [arXiv](https://arxiv.org/abs/1511.06434v1) | [Code]()| [Skimming](https://github.com/doublejy715/Paper_review/issues/30)| [Summary]() |
25 | Pix2pix : Image-to-Image Translation with Conditional Adversarial Networks | CVPR | 2016 | [arXiv](https://arxiv.org/abs/1611.07004) | [Code]()| [Skimming](https://github.com/doublejy715/Paper_review/issues/31)| [Summary]() |
26 | GAN Dissection: Visualizing and Understanding Generative Adversarial Networks | CVPR | 2015 | [arXiv](https://arxiv.org/abs/1811.10597) | [Code]()| [Skimming](https://github.com/doublejy715/Paper_review/issues/32)| [Summary]() |

To Do List
------------
No | Title | Conference | Year | Link | Code | Skimming | Summary |  
------------ | ------------- |----------|----------|----------|----------|----------|----------|  
1 | Image Style Transfer Using Convolutional Neural Network | CVPR, IEEE | 2016 | [IEEE](https://ieeexplore.ieee.org/document/7780634) | [Practice](https://github.com/ShaharAssenheim/Image-Style-Transfer-Using-Convolutional-Neural-Network) || [Summary](https://github.com/doublejy715/Paper_review/issues/6)|
2 | StyleRig : Rigging stylegan for 3d control over portrait images | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2004.00121) | None || [summary]() |
3 | Ganspace : Discovering interpretable gan controls | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2004.02546) | [Code](https://github.com/harskish/ganspace)| | [Summary]() |
4 | In-domain gan inversion for real image editing | CVPR | 2020 | [arXiv](https://arxiv.org/abs/2004.00049) | [Code](https://github.com/genforce/idinvert)| | [Summary]() |
5 | BicycleGAN : Toward Multimodal Image-to-Image Translation | CVPR | 2017 | [arXiv](https://arxiv.org/abs/1711.11586) | [Code](https://github.com/junyanz/BicycleGAN) | |[Summary]() |
6 | MaskGAN : Batter Text Generation Via Filling In the _____ | ICLR | 2018 | [arXiv](https://arxiv.org/abs/1801.07736v1) | None | |[Summary]() |
7 | GANspace |  |  | |  |  ||
8 | PGGAN |  |  | |  |  ||
9 | BigGAN |  |  | |  |  ||
10 | SPADE : Semantic Image Synthesis with Spatially-Adaptive Normalization | CVPR | 2019 | [arXiv](https://arxiv.org/abs/1903.07291) | [Code](https://github.com/NVlabs/SPADE) || [Summary](https://github.com/doublejy715/Paper_review/issues/21) |
11 | Interpreting the latent space of gans for semantic face editing | CVPR | 2020 | [arXiv](https://arxiv.org/abs/1907.10786)| None | | [Summary](https://github.com/doublejy715/Paper_review/issues/20) |
12 | CycleGAN : Unpaired Image-to-Image Translation using Cycle-Consistent Adversarial Networks | CVPR | 2017 | [arXiv](https://arxiv.org/abs/1703.10593v1) | [Code-pytorch](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix) || [Summary](https://github.com/doublejy715/Paper_review/issues/23)
23 | pix2pixHD : High-Resolution Image Synthesis and Semantic Manipulation with Conditional GANs | CVPR | 2017 | [arXiv](https://arxiv.org/abs/1711.11585v1) | [Code](https://github.com/NVIDIA/pix2pixHD) | |[Summary]() |
